docs_src/getting-started/tools/Ch-GUI.md

```diff
@@ -6,14 +6,25 @@ EdgeX's graphical user interface (GUI) is provided for demonstration and develop
 You can quickly run the GUI in a Docker container or as part of the EdgeX Snap.  You can also download, build and run the GUI natively on your host.
 
 ### Docker Compose
-The EdgeX GUI is already incorporated into Docker Compose files provided by the project.  Locate and download the Docker Compose file that includes the GUI container by looking for a file that has "-with-ui" in the file name.  For example, in the Ireland branch of edgex-compose, notice the two Docker Compose files that include the GUI below.
+The EdgeX GUI is now incorporated into all the secure and non-sure Docker Compose files provided by the project.  Locate and download the Docker Compose file that best suits your needs.  For example, in the Jakarta branch of edgex-compose the `*-with-app-sample*` compose files include the Sample App Service allowing the configurable pipeline to be manipulated from the UI. See the four Docker Compose files that include the Sample App Service below.
```

Comment:
where are those `four` docker compose files below? I could not find it.
suggest linking this to the docker compose file example mentioned.

Reply:
They are circled in the image. I have added a link to edgex-compose and text about `circled` below.

Reply:
okay, i see